### ISSUE TITLE
Parallelization 

### ISSUE THREAD

**Issue by gleitz** [gleitz/howdoi#354]
When you ask for multiple answers with `-n5` we should request those in parallel

Q: How do we preserve the order?

This doesn't work but might be close:
```    
from multiprocessing import Pool

def _get_answer_worker(answer_number):
        current_position = answer_number + initial_position
        args['pos'] = current_position
        link = get_link_at_pos(question_links, current_position)
        answer = _get_answer(args, question_links)
        if not answer:
            return ''
        if not args['link'] and not args['json_output'] and multiple_answers:
            answer = ANSWER_HEADER.format(link, answer, STAR_HEADER)
        answer += '\n'
        return answer

    with Pool() as pool:
        result = pool.map(_get_answer_worker, range(args['num_answers']))
        print(result)
```

**Comment by gleitz** [comment]
cc @SaurusXI

**Comment by SaurusXI** [comment]
@gleitz lets close this issue?

**Comment by gleitz** [comment]
Fixed in #357
